build.sh

```diff
@@ -19,7 +22,7 @@ done
 
 REGISTRY_ADDRESS=${d:-divoc}
 REPO=${r:-"https://github.com/egovernments/DIVOC.git"}
-
+etcd=${e:-"127.0.0.1:2379"}
```

Comment:
We can remove this and make ETCD a mandatory field IMO.

Reply:
Probably add a error message saying that this is expected.

Reply:
if we have one single machine to install on, we can skip this. so added a default localhost to etcd